events/equeue/equeue.c

```diff
@@ -602,7 +605,7 @@ static void equeue_chain_dispatch(void *p)
 static void equeue_chain_update(void *p, int ms)
 {
     struct equeue_chain_context *c = (struct equeue_chain_context *)p;
-    equeue_cancel(c->target, c->id);
+    (void)equeue_cancel(c->target, c->id);
```

Comment:
What happens if the cancellation would fail in this case? Is it possible, and should we stop then?

Reply:
I'm not sure so I left it as it is, maybe @geky could help with this one?

Reply:
Failure is ok in this case. If we don't cancel the old chain event the worst thing that happens is our chained equeue gets a spurious wakeup. But equeue handles spurious wakeups by always checking the time and going back to sleep if it's too early.